packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1713,7 +1714,7 @@ abstract class ParentDataWidget<T extends ParentData> extends ProxyWidget {
 abstract class InheritedWidget extends ProxyWidget {
   /// Abstract const constructor. This constructor enables subclasses to provide
   /// const constructors so that they can be used in const expressions.
-  const InheritedWidget({ Key key, Widget child })
+  const InheritedWidget({ Key? key, Widget? child })
```

Comment:
child should be required, non-nullable

Reply:
Some null may be here accros all children.

Reply:
Done